libtenzir/builtins/operators/show.cpp

```
@@ -0,0 +1,451 @@
+              }
+            },
+            [&](const list_type& l) {
+              auto list = structure.push_field("list");
+              auto err = list.add(fmt::to_string(l.value_type()));
+              (void)err;
+            },
+            [&](const map_type& m) {
+              auto map = structure.push_field("map");
+              auto record = map.push_record();
+              auto key_field = record.push_field("key");
+              auto err = key_field.add(fmt::to_string(m.key_type()));
+              auto value_field = record.push_field("value");
+              err = value_field.add(fmt::to_string(m.value_type()));
+              (void)err;
+            },
+            [&](const record_type& r) {
+              auto record = structure.push_field("record");
+              auto list = record.push_list();
+              for (const auto& field : r.fields()) {
+                auto field_record = list.push_record();
+                auto field_name = field_record.push_field("name");
+                auto err = field_name.add(field.name);
+                auto field_type = field_record.push_field("type");
+                err = field_type.add(fmt::to_string(field.type));
+                (void)err;
+              }
+            },
+          };
+          caf::visit(f, type);
+          auto attributes = collect(type.attributes());
+          if (attributes.empty())
+            continue;
+          auto list = row.push_field("attributes").push_list();
+          for (auto& attribute : attributes) {
+            auto record = list.push_record();
+            err = record.push_field("key").add(attribute.key);
+            err = record.push_field("value").add(attribute.value);
+          }
+          (void)err;
+        }
+        co_yield builder.finish();
+      } else {
+        die("invalid catalog interaction");
+      }
+    } else {
+      die("unchecked aspect");
+    }
+  }
+
+  auto name() const -> std::string override {
+    return "show";
+  }
```

I think this clashes with the `show` command plugin, no? This should really cause an error, but for me it silently fails and makes the `show` operator unavailable.

What can we do here to resolve this situation?

---

Can we simply rip out the `show` command? I don't think anyone relies on it, and we're about to port the information over to the corresponding operator anyway.